lib/src/user_location_options.dart

```diff
@@ -11,15 +11,15 @@ class UserLocationOptions extends LayerOptions {
   List<Marker> markers;
   Widget markerWidget;
   StreamController markerlocationStream;
-  bool updateMapOnPositionChange = true;
+  bool updateMapLocationOnPositionChange;
 
   UserLocationOptions({
-    this.context,
-    this.mapController,
-    this.markers,
+    @required this.context,
+    @required this.markers,
+    @required this.markerlocationStream,
```

Comment:
`markerLocationStream`  provides the current location of the marker, this functionality might not be required in all use cases. Let's make this optional parameter